sdk/python/feast/sdk/utils/bq_util.py

```diff
@@ -74,7 +74,8 @@ def get_table_name(feature_id, storage_spec):
     except KeyError:
         raise ValueError("storage spec has empty project or dataset option")
 
-    table_name = "_".join(feature_id.split(".")[:2])
+    print(feature_id)
```

Comment:
I think the `print` is not necessary here.
I guess this was for debugging last time?

Reply:
Nice catch